include/fsutils/ipcfg.h

```diff
@@ -137,7 +227,8 @@ int ipcfg_read(FAR const char *netdev, FAR struct ipcfg_s *ipcfg);
  ****************************************************************************/
 
 #ifdef CONFIG_IPCFG_WRITABLE
-int ipcfg_write(FAR const char *netdev, FAR const struct ipcfg_s *ipcfg);
+int ipcfg_write(FAR const char *netdev, FAR const struct ipcfg_s *ipcfg,
+                sa_family_t af);
```

Comment:
If ipcfg_s already contain type field, why the call stll have to pass af argument?

Reply:
Fixed.  the header is not managed internally by ipcfg_write() and ipcfg_read();

Reply:
is this the classic not->now?